modules/mobile/app/models/mobile/v0/authorized_services.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class AuthorizedServices
+      def initialize(user)
+        @user = user
+      end
+
+      def to_a
+        authorized_services_map.select { |_, v| v == true }.keys
+      end
+
+      def available_services
+        authorized_services_map.keys
+      end
+
+      private
+
```

Comment:
In addition to the policies below, the decision letters controller also uses the bgs policy but isn't in the authorized services. Not sure why.

Reply:
Talk to front end. Can we add it preemptively? Do they need to make changes on their end?